File: examples/key/new_managed_certificate/new_managed_certificate.go
package main

import (
	"github.com/bloock/bloock-sdk-go/v2/client"
	"github.com/bloock/bloock-sdk-go/v2/entity/key"
)

func main() {
	keyClient := client.NewKeyClient()

	_, _ = keyClient.NewManagedCertificate(key.ManagedCertificateParams{
		KeyType: key.Rsa2048,
		Subject: key.SubjectCertificateParams{
			CommonName: "BLOOCK",
		},
		ExpirationMonths: 12,
	})
}

File: examples/key/new_managed_certificate/new_managed_certificate.php
<?php

require "./vendor/autoload.php";

use Bloock\Client\KeyClient;
use Bloock\Entity\Key\KeyType;
use Bloock\Entity\Key\SubjectCertificateParams;
use Bloock\Entity\Key\ManagedCertificateParams;

$keyClient = new KeyClient();

$key = $keyClient->newManagedCertificate(new ManagedCertificateParams(KeyType::Rsa2048, new SubjectCertificateParams("Bloock"), 2));
